src/containers/studies/StudiesSagas.js

```diff
@@ -142,6 +145,50 @@ const LOG = new Logger('StudiesSagas');
 const CAFE_ORGANIZATION_ID = '7349c446-2acc-4d14-b2a9-a13be39cff93';
 const TIME_USE_DIARY = 'Time Use Diary';
 
+function* deleteStudyWorker(action :SequenceAction) :Saga<*> {
+  try {
+    yield put(deleteStudy.request(action.id));
+
+    const study = action.value;
+
+    const studyId = study.getIn([STUDY_ID, 0]);
+    const studyEKID = getEntityKeyId(study);
+
+    yield call(ChronicleApi.deleteStudy, studyId);
```

Comment:
Why aren't you checking `response.error`?

Reply:
No need to explicitly check for errors as they would still be propagated to the `catch` block

Reply:
Right! This is not `lattice-sagas`.